canonical-web-and-design/maas-ui
# Make KVM table responsive.

## Done

- Gave KVM table cells explicit widths and breakpoints at which they disappear/change width.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to /MAAS/r/kvm and check that the table looks pretty good on all viewport sizes, i.e. it doesn't look broken on small screens.

## Fixes

Fixes: canonical-web-and-design/MAAS-squad#2025
